docs/components/input-checkbox.md

```diff
@@ -145,6 +118,15 @@ like:
 .KCheckbox-wrapper {
   --KCheckboxPrimary: blueviolet;
 }
+
+.k-checkbox {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏do we still need this CSS?

Reply:
No! Removed it. Thanks for catching this.